templates/has_joined_cluster.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Skript um einen Test solange auszuführen, bis erfolgreich oder timeout
+
+# Konfig
+# ------
+# maximale Wartezeit in Sekunden
+maxwait={{ mariadb.galera.join.max_wait }}
+# Pause zwischen 2 Versuchen in Sekunden
+pausetime={{ mariadb.galera.join.check_pause }}
+
+starttime=`date +%s`
+endtime=`expr $starttime + $maxwait`
+
+while [ `date +%s` -lt $endtime ]
+do
+        if echo "SHOW STATUS LIKE 'wsrep_ready';" | mysql | grep 'ON'
```

Comment:
We should 4 instead of 8 spaces (like this is done in other roles for config files and scripts).

Reply:
Fixed in 6b3952edf64fc2cd0a815ec9d1cfa04816405f30